### PR TITLE
✨Add Game Page

### LINKED ISSUE #34: Add basic Gameboard page
## Acceptance criteria

- [x] Add basic gameboard page

### PR DESCRIPTION
**Pull request by fre-ben**
- Add Game page
- Add route from Lobbies page to Game page
- Adjust  CSS for some components

The layout isn't perfect yet.
Need to fix the user being able to scroll down, there seems to be some issue with the `gamElements8Player` class.
If anyone generally has any ideas on how to implement the design in a smarter way, I'm open for suggestions :)

Deployment: https://sisu-pipelin-gameboardp-eilheb.herokuapp.com/game
Fixes  #34

### CODE REVIEW

**Review by mewiemers — COMMENTED** [review]
GJ 🎲🎲

File: styles/Game.module.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+.pageElements {
+  display: grid;
+  grid-template-columns: 80% 20%;
+  grid-template-rows: auto 1fr;
+  align-items: start;
+  column-gap: 2em;
+  width: 100%;
+  grid-template-areas:
+    "topbar sidebar"
+    "game sidebar";
```
Comment (mewiemers) [inline styles/Game.module.css:36]: I am not sure, but where is your `game`class , you declared in your grid-template areas?

**Review by RonjaMeinecke — APPROVED** [review]
🦾LGTM

**Review by ghost — APPROVED** [review]
Good job! 🚀

File: styles/Game.module.css (hunk 1 above)
Comment (fre-ben) [reply styles/Game.module.css:36]: Yep, you're right - forgot to set it. Thanks :)

**Pull request merged**